### ISSUE TITLE
ConcatOp : Dimensions of inputs should match

### ISSUE THREAD

**Issue by guanfuchen** [3ammor/Video-Pixel-Networks#2]
![image](https://user-images.githubusercontent.com/22321977/33364731-a2d56c14-d520-11e7-8b39-929ffca26188.png)

```bash
 python vpn.py --vpn_arch='micro' --train=True --overfitting=False --data_dir='mnist_test_seq.npy' --exp_dir='XXX'
```

Thanks for your implementation for VPN. I run the model for about 3hours，but it occurs the error，I do not know how it should be modified，the version of tensorflow I use is cpu_1.0.0:
```bash
InvalidArgumentError (see above for traceback): ConcatOp : Dimensions of inputs should match: shape[0] = [1,64,64,1] vs. shape[1] = [16,64,64,31]
         [[Node: inference_graph/vpn_decoder/pixel_cnn_decoders/residual_multiplicative_block_with_mask_1/concat = ConcatV2[N=2, T=DT_FLOAT, Tidx=DT_INT32, _device="/job:localhost/$
eplica:0/task:0/cpu:0"](_recv_inputs/inference_current_frame_0, inference_graph/vpn_decoder/pixel_cnn_decoders/residual_multiplicative_block_with_mask_1/h1/BiasAdd, inference_graph$
vpn_decoder/pixel_cnn_decoders/residual_multiplicative_block_with_mask_1/concat/axis)]]
```

**Comment by gnosisyuw** [comment]
This error is because the test part is actually generating the predicted frame instead of calculating the loss. So the required input dimension should be [1,width, height, channel]. But the input defined here has shape of [batch_size, width, height, channel].

**Comment by o-tawab** [comment]
Hi @guanfuchen,

Sorry for the very late reply, 

I think your problem should be fixed now.

**Comment by vishal2612200** [comment]
@guanfuchen How did you solve this problem?Can you please explain your solution if you have any.
